### PR TITLE
[13.0][MIG] hr_expense_invoice

### PR DESCRIPTION
**Pull request by Saran440**

### CODE REVIEW

**Review by ps-tubtim — APPROVED** [review]
Code LGTM

**Comment by sotolucas** [comment]
@pedrobaeza who has to merge this?
Thanks in advance.

**Comment by alexey-pelykh** [comment]
@sotolucas once the PR gets enough reviews, it will get "ready to merge" label and a person from PSC team will merge it

File: hr_expense_invoice/__init__.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```
Comment (alexey-pelykh) [inline hr_expense_invoice/__init__.py]: I don't really see value of removing license from these files

**Review by alexey-pelykh — APPROVED** [review]
Code LGTM, minor comments

**Comment by pedrobaeza** [comment]
Please rebase and fix the problems.

**Comment by Saran440** [comment]
Thank you for all reviewed. Fixed :)

**Pull request merged**